mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
+//Getting a random number between 0 (inclusive) and 1 (exclusive)
```

Comment:
```suggestion
//Getting a random number (float) between 0 (inclusive) and 10 (exclusive)
```

Reply:
Math.random() return a number (float) between 0 (inclusive) and 1 (exclusive)
But when we multiply by 10 it will be between 0 and <10

Reply:
> Math.random() return a number (float) between 0 (inclusive) and 1 (exclusive) But when we multiply by 10 it will be between 0 and <10

You are definitely right! :100: 
Thank you so much!